src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString.split(';');
+  const filteredStyles = styles.filter(
+    (styleDeclaration) => styleDeclaration.trim() !== '',
+  );
+
+  const styleObject = {};
+
+  filteredStyles.forEach((style) => {
+    const [property, value] = style.split(':').map((item) => item.trim());
+
+    styleObject[property] = value;
+  });
+
+  return styleObject;
```

Comment:
The function implementation is correct and follows the task description well. However, it's important to note that the code could be made more concise by chaining the `split`, `filter`, and `forEach` methods directly instead of using separate variables. This would also adhere to the checklist item regarding chaining methods with the same indentation level. Additionally, consider using `reduce` instead of `forEach` to build the `styleObject`, as it is more idiomatic for this use case and avoids the need to declare an empty object beforehand.